.storybook/reset.scss

```diff
@@ -7,5 +7,5 @@ body,
 body {
   margin: 0;
   padding: 0;
-  font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
```

Comment:
This isn't going to work reliably without a link to the Google font somewhere. 
Maybe we should just go straight to the `IBM Plex Sans` though?